app/services/topology_service_api.rb

```diff
@@ -0,0 +1,15 @@
+require 'topological_inventory-api-client'
+class TopologyServiceApi
+  attr_accessor :params, :api_instance
+  def initialize(options)
+    @params = options
+    TopologicalInventoryApiClient.configure do |config|
+      # Configure HTTP basic authorization: UserSecurity
+      config.username = 'YOUR USERNAME'
```

Comment:
@Fryguy - Do we have an agreed upon pattern for handling settings? I'm assuming these would need to come in as environment variables for OpenShift, but the right pattern may allow us to break those out by the Rails environment.